aiida/tools/dbexporters/tcod.py

```diff
@@ -200,21 +202,21 @@ def encode_textfield_quoted_printable(content):
     content = quopri.encodestring(content)
 
     def match2qp(m):
-        prefix = ''
-        postfix = ''
+        prefix = b''
+        postfix = b''
         if 'prefix' in m.groupdict().keys():
             prefix = m.group('prefix')
         if 'postfix' in m.groupdict().keys():
             postfix = m.group('postfix')
         h = hex(ord(m.group('chr')))[2:].upper()
         if len(h) == 1:
-            h = "0{}".format(h)
-        return "{}={}{}".format(prefix, h, postfix)
+            h = "0%s" % h
+        return b"%s=%s%s" % (prefix, h.encode('utf-8'), postfix)
```

Comment:
Is there a reason to use % instead of .format()? In all the rest of the code we use .format()

Reply:
(Also for this, same question in many points)

Reply:
@yakutovicha, have you changed these lines? I am actually not aware of the differences between ``%`` and ``format``.

Reply:
> (Also for this, same question in many points)

This seems to be the last occurrence now, I have replaced others.